frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -540,24 +508,16 @@ export const ADMIN_SETTINGS_SECTIONS = {
           },
         },
       },
+
       {
-        key: "-embedded-dashboards",
-        display_name: t`Embedded Dashboards`,
-        widget: EmbeddedDashboardListing,
-        getHidden: (_, derivedSettings) => !derivedSettings["enable-embedding"],
-      },
-      {
-        key: "-embedded-questions",
-        display_name: t`Embedded Questions`,
-        widget: EmbeddedQuestionListing,
+        key: "-embedded-resources",
+        display_name: t`Manage embeds`,
+
+        widget: EmbeddedResources,
```

Comment:
We should start thinking about a lighter way to cache API calls. This component calls APIs directly, so there's no cache, and that makes loading a bit jarring.
https://github.com/metabase/metabase/blob/37a620f214eefbdd50c6352d5284f1cc3bc49542/frontend/src/metabase/admin/settings/components/widgets/PublicLinksListing/PublicLinksListing.jsx#L184-L190

If you go back and fourth to the same settings page, it will still show the loading indicators every time.
![Screenshot 2023-12-25 at 2 21 25 PM](https://github.com/metabase/metabase/assets/1937582/718c6c1f-3858-4443-9f97-187c0b698e0d)